airbyte-integrations/template/singer-source/template_singer_source/source.py

```diff
@@ -0,0 +1,28 @@
+import urllib.request
+from typing import Generator
+
+from airbyte_protocol import AirbyteCatalog
+from airbyte_protocol import AirbyteCheckResponse
+from airbyte_protocol import AirbyteMessage
+from airbyte_protocol import Source
+from base_singer import SingerHelper
+
+
+class TemplateSingerSource(Source):
```

Comment:
This also feels like it shouldn't be based on the exchange rate api. Either we should actually use the exhangerates api as the template we point devs to or we should use something that won't diverge.

Reply:
agreed. Let's take that conversation to the other PR with the singer shim